src/formatDate.js

```diff
@@ -9,6 +9,58 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateSeparated = date.split(fromFormat[3]);
+  let day = '';
+  let month = '';
+  let year = '';
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
What if fromFormat or toFormat has more than 3 elements? To ensure flexible code, it's good to handle scenarios where fromFormat or toFormat might have more than 3 elements. This improves the function's usability and reliability. What I would use is `forEach` loop here.